lua/telescope/builtin/files.lua

```diff
@@ -244,6 +244,12 @@ files.file_browser = function(opts)
         current_picker:refresh(gen_new_finder(new_cwd), { reset_prompt = true })
       end)
 
+      local change_working_directory = function(prompt_bufnr)
```

Comment:
this `prompt_bufnr` shouldn't be needed because we already have one up above. That's the reason why the CI is failing (you could do `make lint` locally to make sure it passes)

Reply:
You are right! my bad. Let me try this again